src/templates/company/index.js

```diff
@@ -34,42 +38,46 @@ export default {
   ],
   optionalModules: [{ appName: 'agent.aragonpm.eth', label: 'Agent' }],
   screens: [
-    [data => completeDomain(data.domain) || 'Claim domain', ClaimDomain],
-    ['Configure template', Voting],
-    ['Configure template', Tokens],
+    [
+      data => completeDomain(data.domain) || 'Claim domain',
+      props => <ClaimDomainScreen screenProps={props} />,
+    ],
+    ['Configure template', props => <VotingScreen screenProps={props} />],
```

Comment:
Curious here, would this work: `['Configure template', VotingScreen]` (with the appropriate changes to receive the props directly instead of `screenProps`)

Reply:
Yes, actually that’s how it used to be :smile: 

It was having two problems: props were passed directly, making it hard to make generic, and some components had to be wrapped so the could be configured (`Tokens`, `Review`). So I decided to move the props into `screenProps`, and make everything consistent (also using `dataKey` so that data is written exactly the same way).